interface/src/app/experiment/experiment.component.js

```diff
@@ -0,0 +1,69 @@
+angular.module('adage.experiment', [
+  'adage.experiment.service',
+  'adage.analyze.sample',   // TODO fix after Sample is also refactored
+  'statusBar'
+])
+
+.component('experimentDetail', {
+  templateUrl: 'experiment/experiment.component.tpl.html',
+  bindings: {
+    id: '<'
+  },
+  controller: [
+    'Experiment', 'Sample', '$log',
+    function(Experiment, Sample, $log) {
+      var ctrl = this;
+
+      ctrl.$onChanges = function(changesObj) {
+        ctrl.show(changesObj.id.currentValue);
+      };
+      ctrl.makeHref = Experiment.makeHref;
+      ctrl.experiment = {
+        status: '',
+        relatedSamples: []
+      };
+
+      var queryError = function(responseObject, responseHeaders) {
```

Comment:
The second argument `responseHeaders` is not needed.

Reply:
True. Thanks!